Modules/mmapmodule.c

```diff
@@ -1034,12 +1034,14 @@ mmap.mmap.flush
     offset: Py_ssize_t = 0
     size: Py_ssize_t = -1
     /
+    flags: int = 0
```

Comment:
I would prefer to only pass flags as a keyword argument, so add `*` in Argument Clinic:


```suggestion
    *
    flags: int = 0
```